src/data/iconData.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Here there should be an icon set for each category.
You could delete all this and when you go to https://openweathermap.org/weather-conditions

You can see it gives you a little explanation how to create icon for each category:
How to get icon URL
For code 500 - light rain icon = "10d". See below a [full list of codes](https://openweathermap.org/weather-conditions#Weather-Condition-Codes-2)
URL is https://openweathermap.org/img/wn/10d@2x.png

You could create something like this:
`````
{
  "200": "https://openweathermap.org/img/wn/11d@2x.png",
  "300": "https://openweathermap.org/img/wn/09d@2x.png",
   ....etc
}
`````